user-guides/message-actions/readme.md

```diff
@@ -0,0 +1,66 @@
+# Message Actions
```

Comment:
This file needs to be added to the Table of Contents https://github.com/RocketChat/docs/blob/master/_data/toc.yml